modules/conflict_resolver/php/NDB_Menu_Filter_resolved_conflicts.class.inc

```diff
@@ -113,21 +112,27 @@ class NDB_Menu_Filter_Resolved_Conflicts extends NDB_Menu_Filter
         );
 
         $this->headers = array(
-                          'TableName',
+                          'Instrument',
                           'CandID',
                           'PSCID',
                           'Visit_label',
-                          'ResolutionTimestamp',
+                          'Timestamp',
```

Comment:
Should be 'Resolution Timestamp' as before. Otherwise it is not clear what it represents